spark/src/test/java/org/apache/zeppelin/spark/SparkSqlInterpreterTest.java

```diff
@@ -161,6 +163,41 @@ public void test_null_value_in_row() {
     assertEquals("name\tage\ngates\tnull\n", ret.message().get(0).getData());
   }
 
+  private SparkInterpreter getSparkInterpreter() {
+    LazyOpenInterpreter lazy = null;
+    SparkInterpreter spark = null;
+    Interpreter p = sql.getInterpreterInTheSameSessionByClassName(SparkInterpreter.class.getName());
+
+    while (p instanceof WrappedInterpreter) {
+      if (p instanceof LazyOpenInterpreter) {
+        lazy = (LazyOpenInterpreter) p;
+      }
+      p = ((WrappedInterpreter) p).getInnerInterpreter();
+    }
+    spark = (SparkInterpreter) p;
+
+    if (lazy != null) {
+      lazy.open();
+    }
+    return spark;
+  }
+
+  @Test
+  public void testVariableInterpolation() {
+    InterpreterContext newContext = new InterpreterContext("NoteId", "ParaId", null,
+            "testVariableInterpolation", "",
+            null, null, null, null,
+            new LocalResourcePool("testVariableInterpolation"),
+            null, null);
+    SparkZeppelinContext zc = getSparkInterpreter().getZeppelinContext();
+    zc.setInterpreterContext(newContext);
+    zc.put("n", "100");
+    zc.put("table", "name");
+    String commandWithVariables = "select * from {table} where count = {n}";
```

Comment:
Great catch. Escaping should be done similarly like in Jupyter.
Since this will be more similar for users coming to Zeppelin from Jupyter world.
With double curly braces –
  {{ will be escaped to single left curly brace - {
  }} will be escaped to single right curly brace - }
Thanks!